### PR TITLE
adding transmitter timestamp format to notes

### PR DESCRIPTION
**Pull request by julian-esteban**

### CODE REVIEW

File: content/api-references/journals.md [hunk 1]
```diff
@@ -120,11 +120,11 @@ For more on Journals click [here]({{< relref "../../integration/funding/#cash-po
...
 | `description`                       | string            | {{<hint info>}}Optional {{</hint>}}   | Max 1024 characters. Can include fixtures for amounts that are above the transaction limit (default = $50). Use only ENUM: `pending` or `rejected`. Journal's status will become this value at the end of the current market day (will default to `executed` if no fixture is inputted). The description can contain fixtures in the following format: `/fixtures/key1=value1,key2=value2/fixtures`. |
-| `transmitter_name`                  | string            | {{<hint info>}}Optional {{</hint>}}   | Max 255 characters. See more details about [Travel Rule]({{< relref "/integration/funding.md#travel-rule" >}}).                                                                                                                                                                                                            |
```
Comment (umitanuki) [inline content/api-references/journals.md:123]: This seems unrelated

Comment (julian-esteban) [reply content/api-references/journals.md:123]: yeah, amending with a commit to remove this change
